strtcpsvr/sc_tcpsvr.c

```diff
@@ -208,16 +195,18 @@ int main(int argc, char *argv[])
     child_argv[1] = "/QOpenSys/pkgs/bin/bash";
     child_argv[2] = "-l";
     child_argv[3] = "-c";
+    int is_it_batch = is_batch();
+    char* batch_prefix_string = (0 == is_it_batch) ? "exec" : "/QOpenSys/usr/bin/nohup";
```

Comment:
I was first a bit confused by the call to nohup (have no experience using this command), but it turns out there actually is a PASE version of nohup.

But there is also a (newer) nohup command in /QOpenSys/pkgs/bin...?

Reply:
Yes. I didn't use `/QOpenSys/pkgs/bin/nohup` because it would require the `cureutils-gnu` package to be installed. 
However, I didn't realize that it's already a requirement for the RPM! I should change it.

Reply:
Changed at https://github.com/ThePrez/ServiceCommander-IBMi/pull/156/commits/0ed59a212e8fe9a840ac744bba24166c984ce557

Thanks!